tools/topology/platform/intel/intel-generic-dmic-kwd.m4

```diff
@@ -42,7 +42,20 @@ ifdef(`DMIC_DAI_LINK_48k_NAME',`',define(DMIC_DAI_LINK_48k_NAME, `dmic01'))
 ifdef(`DMIC_DAI_LINK_16k_NAME',`',define(DMIC_DAI_LINK_16k_NAME, `dmic16k'))
 
 # DMICPROC is set by makefile, available type: passthrough/eq-iir-volume
-ifdef(`DMICPROC', , `define(DMICPROC, passthrough)')
+ifdef(`IGO',
+`define(DMICPROC, nr)',
```

Comment:
should there be a pipe-nr-capture.m4 in the commit, I wonder if it's missing or I just didn't find it. Would the pipeline macro name pipe-igonr-capture.m4 be better?

Reply:
This PR consists of two commits: 
`audio: igo_nr: Added intelliGo noise reduction wrapper code and topology` is the generic codes and topology.
`topology: Added IGO_NR in sof-tgl-max98373-rt5682 topology.` is the customized topology to add igo_nr into that topology

`pipe-nr-capture.m4` is inside the first commit.

Reply:
agree. I will change the name to `pipe-igonr-capture.m4`.